terraform/scripts/create_bootstrap_bundle.sh

```diff
@@ -60,11 +60,11 @@ find_if_exists() {
 # on the machine, the script will exit.
 hab=$(find_if_exists hab)
 hab pkg install core/aws-cli core/coreutils core/gawk core/tar
-hab pkg binlink core/aws-cli aws
-hab pkg binlink core/gawk awk
-# hab pkg binlink core/tar tar
-hab pkg binlink core/coreutils sha256sum
-hab pkg binlink core/coreutils sort
+
+# We add the relevant binaries to the beginning of PATH, rather than
+# binlink, in case we are running on a real Linux system; we don't
+# necessarily want to overwrite system binaries with the binlinks.
+export PATH="$(hab pkg binlink core/aws-cli)/bin":"$(hab pkg path core/gawk)/bin":"$(hab pkg path core/tar)/bin":"$(hab pkg path core/coreutils)/bin":${PATH}
```

Comment:
I think you want `$(hab pkg path core/aws-cli)/bin`, yes?

Reply:
What is the specific breakage this is fixing?

Reply:
@raskchanky :man_facepalming:

Reply:
@chefsalim I ran this on my Linux machine and it didn't work because it couldn't binlink over my system binaries. There's really no reason it needs to permanently overwrite system binaries to do this.

That being said, I think I'll revise this so it binlinks to a separate directory under this script's control and just add _that_ directory to the `PATH`.